### PR TITLE
Update participant and pot table fix

### PR DESCRIPTION
**Pull request by Saidac**
We fixed the input form for participant so that we update the participant name and it gets display in show view for challenge resource.

Pot also gets inserted in the challenge table and shown at the show view.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -166,6 +168,7 @@ app.get('/:uuid', (req, res) => {
 
 app.post('/challenges/:id', (req, res) => {
   db.User.update({
+    name: req.body.participant.name,
```
Comment (izelnakri) [inline index.js]: this object could simply also be req.body.participant right?

Comment (Saidac) [reply index.js]: ... you mean that the .name after req.body.participant is not needed?

Comment (Saidac) [reply index.js]: req.body.participant.name without name : is ok then?

Comment (Saidac) [reply index.js]: req.body.participant.name without name : is ok then?

Comment (Saidac) [reply index.js]: HERES THE REQ BODY Participant
{ id: '2', name: 'William', password: 'asdf' }

**Comment by Saidac** [comment]
Major merge mayhem confusment we dunno why so we actually manually added these changes to the master branch. So lets kill this branch.

**Pull request closed without merging**